examples/supplychain-app/charts/expressapp-quorum/values.yaml

```diff
@@ -34,9 +42,6 @@ expressapp:
   # The environment variables that will store the port to be working on and the end point to ask for requests
   #Eg. apiUrl: api.blockchaincloudpoc.com
   env:
-    ganache_server: http://manufacturer.test.quorum.blockchaincloudpoc.com:15021
```

Comment:
ganache_server needs to be there, just the name should be geth_address.

Reply:
ok , but what should be the env var name for it ?